src/scripts.js

```diff
@@ -179,12 +184,14 @@ radioButtons.forEach(button => {
     button.addEventListener('click', () => {
         if(homeView) {
             allRecipes.innerHTML = ''
+            currentView.innerText = capitalizeFirstLetter(button.value) + " Recipes"
```

Comment:
This isn't working as expected. When on the home page if you click a radio button the inner text says "Favorite (whateverTag) Recipes". We don't want the word favorite here.